mubench.reviewsite/templates/review.phtml

```diff
@@ -432,7 +431,30 @@ $origin_path = $origin_path ? $origin_path : $pathFor('experiment.detector', arr
                     simplemde.codemirror.refresh();
                     document.getElementById("comment-area").style.display="none";
                     <?php endif; ?>
+                    <?php if($user): ?>
+                        var addUrl = "<?= $pathFor('tag.add', array('experiment_id' => $experiment->id,
```

Comment:
I don't like that this is in a dedicated script while the setup for removing tags is inline in the button event handler. I think it should be consistent for both cases. Since its not too much code, I would probably inline it, but I don't really have a strong preference here.